frontends/web/src/components/TaskOwnerPageComponents/Advanced.js

```diff
@@ -43,35 +47,29 @@ const Advanced = (props) => {
                         <Row>
                           <Form.Label column>Task Configuration</Form.Label>
                           <Col sm="12">
-                            <Form.Text
-                              id="paramsHelpBlock"
-                              muted
-                              className="border-bottom"
-                            >
+                            <Form.Text id="paramsHelpBlock" muted>
                               <span style={{ color: "red" }}>BETA Notice</span>:
-                              Once this task has been activated, the task
-                              configuration can no longer be changed.
+                              {props.task.active
+                                ? " This task has been activated. Therefore, "
+                                : " Once this task has been activated, "}
+                              the task configuration can no longer be changed
+                              except for the following properties:
+                              <ul>
+                                <li>
+                                  aggregation_metric.constructor_args.default_weights
```

Comment:
maybe we should indicate that this should be a dict i.e this line can read:
`aggregation_metric.constructor_args.default_weights (this should be a dict)`